bazel/repository_locations.bzl

```diff
@@ -283,79 +283,118 @@ REPOSITORY_LOCATIONS = dict(
     ),
     com_llvm_clang_15 = dict(
         sha256 = "a275cde426a790cdf6eaecc2a33bac320ee61ceb4f5e76d2c73d3c555fdcedfc",
-        urls = ["https://storage.googleapis.com/pixie-dev-public/clang/15.0-pl11/clang-min-15.0-pl11.tar.gz"],
+        urls = [
+            "https://github.com/pixie-io/dev-artifacts/releases/download/clang/15.0-pl11/clang-min-15.0-pl11.tar.gz",
+            "https://storage.googleapis.com/pixie-dev-public/clang/15.0-pl11/clang-min-15.0-pl11.tar.gz",
```

Comment:
shouldn't these be `clang%2F15.0-pl11`?
i.e. escaped `/` in the release name?

Reply:
Both links work, and imo its nicer to not worry about the escape, but if you think they'll support the escaped link longer than the unescaped one, I can change it.

Reply:
Will escape